src/main/java/seedu/eylah/Eylah.java

```diff
@@ -0,0 +1,88 @@
+package seedu.eylah;
+
+import java.util.Scanner;
+import java.util.logging.Logger;
+
+import seedu.eylah.commons.core.LogsCenter;
+import seedu.eylah.commons.core.Version;
+import seedu.eylah.diettracker.logic.Logic;
+import seedu.eylah.diettracker.logic.LogicManager;
+import seedu.eylah.diettracker.logic.commands.CommandResult;
+import seedu.eylah.diettracker.logic.commands.exceptions.CommandException;
+import seedu.eylah.diettracker.logic.parser.exceptions.ParseException;
+import seedu.eylah.diettracker.model.Model;
+import seedu.eylah.diettracker.model.ModelManager;
+
+/**
+ * The main entry for the EYLAH.
+ */
+public class Eylah {
```

Comment:
Just a suggestion, but should we capitalise this, i.e EYLAH?

Reply:
@ailanthustng I think then it might fail style checks and would not conform to Java formatting standards.

Reply:
Yap, that the reason i only capitalize first character